app/controllers/spree/paypal_controller.rb

```diff
@@ -212,6 +189,33 @@ def payment_details(items)
       end
     end
 
+    def itemized_contents(order)
+      items = order.line_items.map(&method(:line_item))
+
+      tax_adjustments = order.adjustments.tax.additional
+      shipping_adjustments = order.adjustments.shipping
+
+      order.adjustments.eligible.each do |adjustment|
+        next if (tax_adjustments + shipping_adjustments).include?(adjustment)
```

Comment:
We should probably do this computation once before the loop instead of doing it for each adjustment. Or even better:
```rb
adjustments_missing_in_items = order.adjustments.eligible - tax_adjustments - shipping_adjustments
```

And then iterate over them. :sunglasses: